doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -337,40 +311,19 @@ Requests to the environment
 
          SearchHead (@eq bool).
 
-   .. cmdv:: SearchHead @term inside {+ @qualid }
-
-      This restricts the search to constructions defined in the modules named
-      by the given :n:`qualid` sequence.
-
-   .. cmdv:: SearchHead @term outside {+ @qualid }
-
-      This restricts the search to constructions not defined in the modules
-      named by the given :n:`qualid` sequence.
-
    .. exn:: Module/section @qualid not found.
```

Comment:
This is the correct error message (the one appearing in the description of `Search` did not exist). But it should only be documented once (in the description of `Search`, more precisely in the description of the `inside` / `outside` clauses.

Reply:
Deleted it here.